src/CalcManager/CalcManager.vcxproj

```diff
@@ -303,6 +303,10 @@
     <ClInclude Include="Ratpack\ratconst.h" />
     <ClInclude Include="Ratpack\ratpak.h" />
     <ClInclude Include="UnitConverter.h" />
```

Comment:
`src/CalcManager/CalculatorManager.cpp` needs `<climits>` include. I can't comment on that file.

Reply:
Fixed by b3694fa